tests/jax/systems/mappo_system_test.py

```diff
@@ -54,36 +63,36 @@ def test_except_trainer(
 
     # Optimizer.
     optimizer = optax.chain(
-        optax.clip_by_global_norm(40.0), optax.scale_by_adam(), optax.scale(-1e-4)
+        optax.clip_by_global_norm(40.0),
+        optax.adam(1e-4),
     )
 
     # Build the system
     test_full_system.build(
         environment_factory=environment_factory,
         network_factory=network_factory,
         logger_factory=logger_factory,
-        checkpoint_subpath=checkpoint_subpath,
+        experiment_path=checkpoint_subpath,
         optimizer=optimizer,
-        executor_parameter_update_period=20,
-        multi_process=False,
-        run_evaluator=True,
+        executor_parameter_update_period=1,
+        multi_process=True,
+        run_evaluator=False,
         num_executors=1,
         use_next_extras=False,
-        sample_batch_size=2,
+        sample_batch_size=5,
+        checkpoint=True,
```

Comment:
What is the thinking behind these variable choices i.e. including checkpointing (Tf does not) and also excluding the evaluator? Links to my next comment

Reply:
Also, I get that we matched the way we were doing so with the TF system tests but is it not necessary to test more than the training node/ train step? Like for example, if other nodes in the system are working as expected?

Reply:
Agreed. We can maybe remove the checkpointer if we are not checking that the files are getting created.

Reply:
Okay I will remove it now